source/producers/ESSStream/KafkaPlugin.cpp

```diff
@@ -78,6 +89,13 @@ KafkaConfigPlugin::KafkaConfigPlugin()
   add_definition(root);
 }
 
+void KafkaConfigPlugin::getEnvironmentSettings() {
+  auto var = getenv("DAQUIRI_KAFKA_BROKER");
```

Comment:
Based on the `getenv` documentation, it is possible to have `var` be an empty string which would then be a bit of a problem. I guess that this is a bit unlikely to happen in most use cases though.